### ISSUE TITLE
Is there any way to specify min connections?

### ISSUE THREAD

**Issue by atrauzzi** [yandex/odyssey#178]
We're troubleshooting an issue right now where we think that occasionally odyssey will initialize a new connection due to load.  Unfortunately, when this happens, the request that caused the connection to initialize is stuck paying the performance penalty of connection startup.

If we configure odyssey for example 20 connections, is there any way to have odyssey open all 20 of them at start so that they are warm?

**Comment by x4m** [comment]
@atrauzzi nope, but you can create such a setting. Just copy-pase everything from pool_ttl and modify exiration in od_router_expire_cb(). Also you can find how to open new connections in od_frontend_attach().

**Comment by atrauzzi** [comment]
I don't think I want to get into maintaining a fork.  Is this not something that could be added as a feature?

**Comment by x4m** [comment]
Sure, we want this feature.

**Comment by atrauzzi** [comment]
Nice.  Well, unfortunately I'm not in any way competent with C, so I doubt you'd even want me making this! :sweat_smile:

At least this issue is open here now and if you or someone else who is familiar enough can, they can add it.

**Comment by rkhapov** [comment]
https://pg-odyssey.tech/configuration/rules.html#min_pool_size
